src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, OnInit, Input, EventEmitter, Output, OnChanges } from '@angular/core';
+import { Observable } from 'rxjs';
+import { take } from 'rxjs/operators';
+
+import { DotLicenseService } from '@services/dot-license/dot-license.service';
+import { DotMessageService } from '@services/dot-messages-service';
+import { DotRenderedPageState, DotPageMode } from '@portlets/dot-edit-page/shared/models';
+
+@Component({
+    selector: 'dot-edit-page-toolbar',
+    templateUrl: './dot-edit-page-toolbar.component.html',
+    styleUrls: ['./dot-edit-page-toolbar.component.scss']
+})
+export class DotEditPageToolbarComponent implements OnInit, OnChanges {
+    @Input()
+    pageState: DotRenderedPageState;
+
+    @Output()
+    cancelToolbar = new EventEmitter<boolean>();
+
+    @Output()
+    reload = new EventEmitter<boolean>();
```

Comment:
This should be renamed to something like: `actionFired` because we shouldn't name an event based on what happen when the event is trigger.